app.py

```diff
@@ -0,0 +1,1111 @@
+                'fuel': [df_non_gps_m['fuel'][i] for i in range(0, 60)],
+                'battery': [df_non_gps_m['battery'][i] for i in range(0, 60)],
+            },
+            'hour_data_0': {
+                'elevation': [df_non_gps_h_0['elevation'][i] for i in range(0, 60)],
+                'temperature': [df_non_gps_h_0['temperature'][i] for i in range(0, 60)],
+                'speed': [df_non_gps_h_0['speed'][i] for i in range(0, 60)],
+                'latitude': ["{0:09.4f}".format(df_gps_h_0['lat'][i]) for i in range(0, 60)],
+                'longitude': ["{0:09.4f}".format(df_gps_h_0['lon'][i]) for i in range(0, 60)],
+                'fuel': [df_non_gps_h_0['fuel'][i] for i in range(0, 60)],
+                'battery': [df_non_gps_h_0['battery'][i] for i in range(0, 60)],
+            },
+            'minute_data_0': {
+                'elevation': [df_non_gps_m_0['elevation'][i] for i in range(0, 60)],
+                'temperature': [df_non_gps_m_0['temperature'][i] for i in range(0, 60)],
+                'speed': [df_non_gps_m_0['speed'][i] for i in range(0, 60)],
+                'latitude': ["{0:09.4f}".format(df_gps_m_0['lat'][i]) for i in range(0, 60)],
+                'longitude': ["{0:09.4f}".format(df_gps_m_0['lon'][i]) for i in range(0, 60)],
+                'fuel': [df_non_gps_m_0['fuel'][i] for i in range(0, 60)],
+                'battery': [df_non_gps_m_0['battery'][i] for i in range(0, 60)],
+            },
+            'hour_data_1': {
+                'elevation': [df_non_gps_h_1['elevation'][i] for i in range(0, 60)],
+                'temperature': [df_non_gps_h_1['temperature'][i] for i in range(0, 60)],
+                'speed': [df_non_gps_h_1['speed'][i] for i in range(0, 60)],
+                'latitude': ["{0:09.4f}".format(df_gps_h_1['lat'][i]) for i in range(0, 60)],
+                'longitude': ["{0:09.4f}".format(df_gps_h_1['lon'][i]) for i in range(0, 60)],
+                'fuel': [df_non_gps_h_1['fuel'][i] for i in range(0, 60)],
+                'battery': [df_non_gps_h_1['battery'][i] for i in range(0, 60)],
+            },
+            'minute_data_1': {
+                'elevation': [df_non_gps_m_1['elevation'][i] for i in range(0, 60)],
+                'temperature': [df_non_gps_m_1['temperature'][i] for i in range(0, 60)],
+                'speed': [df_non_gps_m_1['speed'][i] for i in range(0, 60)],
+                'latitude': ["{0:09.4f}".format(df_gps_m_1['lat'][i]) for i in range(0, 60)],
+                'longitude': ["{0:09.4f}".format(df_gps_m_1['lon'][i]) for i in range(0, 60)],
+                'fuel': [df_non_gps_m_1['fuel'][i] for i in range(0, 60)],
+                'battery': [df_non_gps_m_1['battery'][i] for i in range(0, 60)],
+            }
+
+        }),
+        # For the case no components were clicked, we need to know what type of graph to preserve
+        dcc.Store(id='store-data-config', data={
+            'data_type': '',
```

Comment:
```suggestion
            'parameter': '',
```

`data_type` might be a bit misleading, since it has a meaning already (string, int, etc.)